src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,38 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string,
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+interface State {
+  searchedMovies: Movie[]
+}
+
+export class MoviesList extends React.Component<Props, {}> {
+  state: State = {
+    searchedMovies: this.props.movies,
+  };
+
+  visibleMovies = () => {
+    const { query } = this.props;
+
+    const visibleMovies = [...this.state.searchedMovies].filter(movie => (
+      movie.title.toLowerCase().includes(query.toLowerCase().trimRight())
+      || movie.description.toLowerCase().includes(query.toLowerCase().trimRight())
+    ));
+
```

Comment:
You write a variable twice, so you can create a new variable and assign it to its query.toLowerCase().trimRight(), and you will shorten your line

Reply:
You're right. Thanks for the suggestion!)